tests/Hilke.KineticConvolution.Tests/DirectionExtensionsTests.cs

```diff
@@ -29,26 +29,26 @@ public void When_Direction_Is_Given_Then_BelongsTo_Should_Return_Expected_Result
         [Test]
         public void When_Direction_Is_Close_To_Range_Boundary_Then_BelongsTo_Should_Return_Correct_Result()
```

Comment:
Dans les autres tests, c'est _Expected_ qui est utilisé, plutôt que _Correct_. À changer ici ?

```suggestion
        public void When_Direction_Is_Close_To_Range_Boundary_Then_BelongsTo_Should_Return_Expected_Result()
```

Reply:
done